bin/nim-gdb.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Exit if anything fails
+set -e
+
+which nim > /dev/null || (echo "nim not in PATH"; exit 1)
+which gdb > /dev/null || (echo "gdb not in PATH"; exit 1)
+which readlink > /dev/null || \
+    which greadline > /dev/null || \
+    ([[ "$(uname -s)" == "Darwin" ]] && \
+        (echo "readlink not in PATH. Please install coreutils from homebrew."; exit 1)) || \
+    (echo "readlink not in PATH."; exit 1)
+
+if [[ "$(uname -s)" == "Darwin" ]]; then
+  READLINK=greadlink
+else
+  READLINK=readlink
+fi
```

Comment:
Why don't you do this?
```bash
if [[ "$(uname -s)" == "Darwin" ]]; then
  READLINK=greadlink
  which readlink > /dev/null || (echo "readlink not in PATH."; exit 1)
else
  READLINK=readlink
  which greadlink > /dev/null || (echo "readlink not in PATH. Please install coreutils from homebrew."; exit 1))
fi
```
I think it is easy to understand and easier to maintain that the complex multiline statement from line 8.

Reply:
Yep, I agree. Done